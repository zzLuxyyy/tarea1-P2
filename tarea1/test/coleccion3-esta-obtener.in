crearColeccion

crearFecha 01/01/1993
crearLibro 214589721 "Las aventuras del sapo Ruperto" "Roy" "Berocay" "Un simpático sapo uruguayo vive divertidas y valientes aventuras" 9
agregarEnColeccion

crearFecha 01/09/2005
crearLibro 999001122 "Nunca me abandones" "Kazuo" "Ishiguro" "Jóvenes clonados enfrentan su destino en un mundo distópico" 2
agregarEnColeccion

crearFecha 08/06/1949
crearLibro 123456789 "1984" "George" "Orwell" "Un régimen totalitario controla hasta los pensamientos de sus ciudadanos" 2
agregarEnColeccion

crearFecha 29/07/1954
crearLibro 181920212 "El señor de los anillos: La comunidad del anillo" "J.R.R." "Tolkien" "Una épica aventura para destruir un anillo de poder" 6
agregarEnColeccion

estaEnColeccion 5
estaEnColeccion 214589721
estaEnColeccion 111111111
estaEnColeccion 999001122

obtenerDeColeccion 214589721

liberarColeccion
Fin
