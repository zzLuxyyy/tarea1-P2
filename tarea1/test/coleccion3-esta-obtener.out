1>2>3>4>5>6>7>8>9>10>11>12>13>14>El libro con isbn 5 NO está en el coleccion.
15>El libro con isbn 214589721 está en el coleccion.
16>El libro con isbn 111111111 NO está en el coleccion.
17>El libro con isbn 999001122 está en el coleccion.
18>Libro 214589721
Título: Las aventuras del sapo Ruperto
Autor: Roy Berocay
Descripción: Un simpático sapo uruguayo vive divertidas y valientes aventuras
Género: 9
Fecha de edición: 1/1/1993
19>20>Fin.
